src/osd/PeeringState.cc

```diff
@@ -1703,9 +1703,6 @@ void PeeringState::calc_replicated_acting(
       acting_backfill->insert(up_cand);
       ss << " osd." << i << " (up) accepted " << cur_info << std::endl;
     }
-    if (want->size() >= size) {
```

Comment:
@liewegas it's wrong to break here as we still want all up peers go to the acting_backfill set..